### DISCUSSION TITLE
debug logs for copilot?

### DISCUSSION THREAD

**Discussion by mellery451** [CopilotC-Nvim/CopilotChat.nvim#289] [Q&A] [answered]
is there any way to enable debug logging for the copilot chats? e.g. I guess I'm interested in the various details of request/response going on from client to server. thanks.

**Comment by mellery451** [comment]
RTFM - debug = True in config is what I want, although I'd like to send them to a file or buffer instead of :messages. I'll see if I can figure that out.

**Reply by deathbeam** [reply, marked as answer]
It already sends the logs to file tho, see :copilotchatdebuginfo
